### PR TITLE
Refactor method of re-rendering Exposure History screen on focus

### PR DESCRIPTION
**Pull request by devinjameson**
Why: prior to this commit, we were using `navigation.addListener` to check if the Exposure History screen was focused and updating state to trigger a re-render.

This commit:
- Uses `useIsFocused()` to trigger a re-render of the Exposure History screen on focus
- Removes the call to `observeExposures()` on the Exposure History Stack because it doesn't seem to be doing anything at the moment, given that we are not rendering a badge on the exposure history tab icon

### CODE REVIEW

**Review by aledustet — APPROVED** [review]
🚢  :shipit:

File: src/navigation/ExposureHistoryStack.tsx [hunk 1]
```diff
@@ -20,24 +18,6 @@ const SCREEN_OPTIONS = {
 }
 
 const ExposureHistoryStack: FunctionComponent = () => {
-  const navigation = useNavigation()
-  const { observeExposures } = useExposureContext()
```
Comment (johnschoeman) [inline src/navigation/ExposureHistoryStack.tsx:24]: should we also remove this from the context?

**Review by johnschoeman — APPROVED** [review]
LGTM

File: src/navigation/ExposureHistoryStack.tsx (hunk 1 above)
Comment (devinjameson) [reply src/navigation/ExposureHistoryStack.tsx:24]: I think there is a lot of cleanup that can be done here to remove unused code, but I'll open another PR for that.

**Pull request merged**